tests/docker/config/construct-query.sparql

```diff
@@ -606,14 +607,16 @@ CONSTRUCT {
       } .
     } .
 
+    BIND( BNODE() AS ?simulationParameterBN ) .
+
     OPTIONAL {
-      ?id bmo:attrs ?attrs .
+      ?id nsg:parameter / bmo:attrs ?attrs .
       ?attrs  ?attrs_prop_orig ?attrs_value .
       BIND (IRI(replace(str(?attrs_prop_orig), str(bmo:), str(:)))  AS ?attrs_prop)
     } .
 
     OPTIONAL {
-      ?id bmo:coords ?coords .
+      ?id nsg:parameter / bmo:coords ?coords .
       ?coords  ?coords_prop_orig ?coords_value .
       BIND (IRI(replace(str(?coords_prop_orig), str(bmo:), str(:)))  AS ?coords_prop)
     } .
```

Comment:
Can we do
```sparql
?id nsg:parameter ?parameter
?parameter bmo:attrs ?attrs
```
And then use ?parameter in the construct to avoid blank nodes? Similar question for `?coords`